timescaledb/how-to-guides/backup-and-restore/timescaledb-backup.md

```diff
@@ -27,9 +27,26 @@ get`, or download a binary from
     *   `pg_dump`
     *   `pg_dumpall`
     *   `pg_restore`
-1.  Install `timescaledb-backup`:
+1.  Download `timescaledb-backup`:
     ```bash
-    $ go get github.com/timescale/timescaledb-backup/
+    $ go get -d github.com/timescale/timescaledb-backup/
+    # Ignore any can't load packages errors, if any.
```

Comment:
```suggestion
```

Reply:
Thanks for the suggestions. I'm not sure if I'm including them correctly. Can you confirm that the above two suggestions have been accepted by me?

Reply:
If you go to the `Files Changed` tab in the GH web interface, you can click `Add suggestion to batch` for each suggestion that you want to include. Then you can click `Commit suggestions` at the top of the page when. you're ready. That creates a new commit that you can see in the `Commits` tab.